Makefile

```diff
@@ -32,15 +32,15 @@ build_worker:
 
 docker: $(DOCKERS)
 
-docker_pki_init: build_pki_init
+docker_pki_init: build_pki_init build_pki
```

Comment:
change build_pki to build_pki_setup to be more precise

Reply:
ok